scripts/generate_static_files.ts

```diff
@@ -3,11 +3,12 @@ import path from "path";
 import fs from "fs";
 import { getFeed as whatsNewFeed } from "../utils/releaseNotes";
 import { getFeed as blogFeed } from "../utils/blogPosts";
-import { getSitemapStream } from "../utils/sitemap";
+import { getPublicSitemap, getSitemapStream } from "../utils/sitemap";
 
 async function build() {
   const feeds = writePublic(await feedContent());
   const sitemap = writePublic(await sitemapContent());
+  const publicSitemap = writePublic(await jsonSitemap());
   return Promise.all([feeds]);
```

Comment:
It's unclear to me why we have `sitemap` here, but not included in the `Promise.all` @evantahler